deb-x64/Dockerfile

```diff
@@ -94,14 +93,18 @@ RUN curl -LO https://ftp.gnu.org/gnu/gettext/gettext-${GETTEXT_VERSION}.tar.xz \
     && rm -rf "gettext-${GETTEXT_VERSION}"
 
 # Now install a recent enough liblzma (5.2+) which supports parallel compression
-RUN curl -LO https://github.com/tukaani-project/xz/releases/download/v${LIBLZMA_VERSION}/xz-${LIBLZMA_VERSION}.tar.xz \
-    && echo "${LIBLZMA_SHA256} /xz-5.2.11.tar.xz" | sha256sum --check \
-    && tar -xf /xz-${LIBLZMA_VERSION}.tar.xz \
-    && cd xz-${LIBLZMA_VERSION} \
+RUN if [ "$DD_TARGET_ARCH" = "armhf" ] ; then \
+    LIB_DEST="arm-linux-gnueabihf"; \
+    else \
+    LIB_DEST="aarch64-linux-gnu"; \
```

Comment:
Why do we now have reference to `aarch64` in the `x64` build image?

Reply:
Hmm if I understand correctly. I think you just copy pasted what we had in the `arm64` dockerfile, right? Probably need some changes

Reply:
Because I wrongfully copy pasted the same block as I forgot there was an arm specificity :facepalm: